test/fluree/crypto/jws_test.cljc

```diff
@@ -0,0 +1,21 @@
+(ns fluree.crypto.jws-test
+  (:require [fluree.crypto :as crypto]
+            [clojure.test :as t :refer [deftest testing is]]))
+
+(def kp
+  {:private "42827e1ee6580a3cd367f31c4af2528db7269b8ea30c6cdff0af6e52d0c4480a"
+   :public "03ef89c5add9879110a18f107fe0f71879af36296f2984040d9b2816958d22fbab"})
+
+(deftest jws
+  (let [s   "abcdefg"
+        jws (crypto/create-jws s (:private kp))]
+    (is (= (str "eyJhbGciOiJFUzI1NkstUiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19."
+                "YWJjZGVmZw."
+                #?(:clj
```

Comment:
Will signatures created on cljs/js be verifiable on clj and vice versa?

Reply:
Yes, they are cross-compatible. I've added a test that verifies that behavior.